src/app/view/components/login/login-page-routing.module.ts

```diff
@@ -0,0 +1,35 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { LoginPageResolver } from '@components/login/login-page-resolver.service';
+
+const routes: Routes = [
```

Comment:
If I open login I do a request... and then if a click "forgot password" I do the same request again, right?

Reply:
yes, fixed now.